### PR TITLE
PLATEXP-2986: Add new card

### PR DESCRIPTION
**Pull request by asmita32675**
https://rackspace.atlassian.net/browse/PLATEXP-2986

@ssalinas24 and @Rohi2305 Please review. As per request new card is added.
Let me know if any changes required.

### CODE REVIEW

File: static/_redirects [hunk 1]
```diff
@@ -7,6 +7,12 @@
 # Customer Service
 /docs/customer-service/v1/*      https://docs-customer-service.netlify.app/docs/customer-service/v1/:splat         200
 
+# Rackspace SDDC Business Handbook
+/docs/sddc/sddc-business-handbook/*   https://docs.rackspace.com/docs/sddc/sddc-business-handbook/   200
+
+# Rackspace SDDC Enterprise Handbook
+/docs/sddc/sddc-enterprise-handbook/* https://docs.rackspace.com/docs/sddc/sddc-enterprise-handbook/   200
+
```
Comment (ssalinas24) [inline static/_redirects]: Did you test the links and the UI experience before adding these redirects? Did they not work and is that why you had to add these?

I added a redirect on last week down below on lines 94 or 100 of this same file with:
```
# Private Cloud SDDC Docs
/docs/sddc/*    https://docs-private-cloud-sddc.netlify.app/docs/sddc/:splat    200
```

Comment (asmita32675) [reply static/_redirects]: When I click on card link it is showing 404 error. So, thought of updating redirect file. But still not working.

Comment (asmita32675) [reply static/_redirects]: Now links are working.

File: layouts/partials/components/docs-content.html [hunk 2]
```diff
@@ -610,6 +610,23 @@ <h4 id="docs-microsoft-hyper-v">Rackspace Server Virtualization by Microsoft Hyp
...
+                                    <div class="list-column">
+                                        <ul>
+                                            <li><a href="/docs/sddc/sddc-business-handbook/">SDDC - Business Handbook</a></li>
+                                            <li><a href="/docs/sddc/sddc-enterprise-handbook/">SDDC - Enterprise Handbook</a></li>
+                                        </ul>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
```
Comment (ssalinas24) [inline layouts/partials/components/docs-content.html]: I believe Lane mentioned to me that this should be the first card on top of the first column. So I say lets move it up to top of its column.

**Review by ssalinas24 — CHANGES_REQUESTED** [review]

File: layouts/partials/components/docs-content.html [hunk 3]
```diff
@@ -609,7 +626,7 @@ <h4 id="docs-microsoft-hyper-v">Rackspace Server Virtualization by Microsoft Hyp
                                 </div>
                             </div>
                         </div>
-                    </div>
+                    </div>
```
Comment (ssalinas24) [inline layouts/partials/components/docs-content.html]:
```suggestion
                    </div>
```

**Review by ssalinas24 — APPROVED** [review]
LGTM
Good work! I can see the links working in the deploy preview now.

**Pull request merged**